domain/src/main/kotlin/com/krzykrucz/elesson/currentlesson/shared/CurrentLessonSharedKernel.kt

```diff
@@ -56,3 +59,16 @@ data class LessonHourNumber private constructor(val number: NaturalNumber) {
 
 data class Teacher(val firstName: FirstName,
                    val secondName: SecondName)
+
+sealed class CurrentLesson // TODO add the rest of lessons
```

Comment:
//TODO left, when will it be implemented?

Reply:
same reply as in CurrentLessonDatabaseAdapter.kt applies